docs/examples/queries/shop.md

```diff
@@ -0,0 +1,133 @@
+---
+id: shop
+title: Shop
```

Comment:
I think the `title` should be more descriptive, like "Shop Info, Goals, and Order Messages"
and then add a `sidebar_label` like "Shop Info"